cli/app/app.go

```diff
@@ -1207,10 +1214,27 @@ func print(name, trailing string, data []byte, toStdout, generateYaml bool, f *o
 	}
 }
 
-// Up brings up rc, svc.
+func validateK8S(client *client.Client) (error){
+	_, err := client.Services(api.NamespaceSystem).List(api.ListOptions{})
+	return err
+}
+
+// Up brings up deployment, svc.
 func Up(c *cli.Context) {
-	fmt.Println("We are going to create deployment controller and service for your dockerized application. \n" +
-		"If you need more kind of controllers, consider to use kompose convert and kubectl. \n")
+	fmt.Println("We are going to create Kubernetes deployment and service for your dockerized application. \n" +
+		"If you need more kind of controllers, use 'kompose convert' and 'kubectl create -f' instead. \n")
+
+	factory := cmdutil.NewFactory(nil)
+	clientConfig, err := factory.ClientConfig()
+	if err != nil {
+		logrus.Fatalf("Failed to get config for accessing the Kubernetes cluster: %v", err)
+	}
+	client := client.NewOrDie(clientConfig)
+
+	//FIXME: try to list out system services in order to check whether k8s is running or not
+	if err := validateK8S(client); err != nil {
```

Comment:
You don't need this, if `factory.ClientConfig()` returns no error, k8s is running.